README.md

```diff
@@ -2,221 +2,62 @@ Category
 
-* Jumper Pin 1-2 = DIP switch 1
-* Jumper Pin 3-4 = DIP switch 2
-* JUmper Pin 5-6 = DIP switch 3
+* Jumper Pin 1-2 = DIP switch 1  (AutoBoot)
+* Jumper Pin 3-4 = DIP switch 2  (Recovery Mode)
+* JUmper Pin 5-6 = DIP switch 3  (Fastboot Mode)
 
-Rescue Using hisi_idt
-=====================
+Simple Rescue Flashing
+======================
 
-This document descirbes the steps to use hisi-idt.py to download binaries to HiKey960 through serial port. On HiKey960, serial port means a ttyUSB device emulated from its OTG port.
+This document descirbes the steps to rescue flash the base firmware for the HiKey960 through the OTG port.
 
-1. Command syntax
------------------
 
-```sh
-sudo ./hikey_idt -c config -p /dev/ttyUSB1
-```
-
-2. Download to DDR
-------------------
+1. Set the board to recovery/forced-download mode:
+--------------------------------------------------
 
-* In the release package, please find these files:
-  - sec_usb_xloader.img
-  - sec_uce_boot.img
-  - sec_fastboot.img
-* Change Jumper settings, to enter forced-download mode:
-  - find J2001, short Pin 1-2
-  - find J2001, short Pin 3-4
+* Remove power from the board
+* Change Jumper settings, to enter recovery/forced-download mode:
+  - find J2001, short Pin 1-2/DIP switch 1 ON
+  - find J2001, short Pin 3-4/DIP switch 2 ON
 * Apply power to the board.
 * Insert USB Type-C cable (OTG port) to the board, and connect the other end to your Linux PC, then
 * Check whether there is a device node "/dev/ttyUSBx". If there is, it means your PC has detected the target board; If there is not, try to repeat previous steps.
-* Prepare the config file. The third image runs on Cortex A53 core, and others are not. You need to put all images into the same directory of hikey_idt. The contents should be in below.
-  - ./sec_usb_xloader.img 0x00020000
-  - ./sec_uce_boot.img 0x6A908000
-  - ./sec_fastboot.img 0x1AC00000
-* Run the script. Eg. if the device node you get from step e is "/dev/ttyUSB1", then use "-d /dev/ttyUSB1". A complete command line looks like:
-```sh
-$ sudo ./hikey_idt -c config -p /dev/ttyUSB1
-```
-* Once images are downloaded successfully, it will print out below log:
-```sh
-Config name: config
-Port name: /dev/ttyUSB1
-0: Image: ./sec_usb_xloader.img Downalod Address: 0x20000
-1: Image: ./sec_uce_boot.img Downalod Address: 0x6a908000
-2: Image: ./sec_fastboot.img Downalod Address: 0x1ac00000
-Serial port open successfully!
-Start downloading ./sec_usb_xloader.img@0x20000...
-file total size 104256
-downlaod address 0x20000
-Finish downloading
-Start downloading ./sec_uce_boot.img@0x6a908000...
-file total size 24000
-downlaod address 0x6a908000
-Finish downloading
-Start downloading ./sec_fastboot.img@0x1ac00000...
-file total size 3110912
-downlaod address 0x1ac00000
-Finish downloading
-```
 
-3. Burn Images
---------------
+2. Flash the recovery image
+---------------------------
```

Comment:
Add a note:

Note: suppose the device node you get in step one is '/dev/ttyUSB1'